tests/unit/mesh/test_ncmesh.cpp

```diff
@@ -1690,6 +1690,64 @@ TEST_CASE("Parallel RP=I", "[Parallel], [NCMesh]")
    }
 }
 
+TEST_CASE("InternalBoundaryProjectBdrCoefficient", "[Parallel], [NCMesh]")
```

Comment:
I copied this test to the `master` branch, and the unit tests all passed. Should this test what was broken before this PR? Should there be a nonconforming refinement in this test?

Reply:
You're correct, I updated the test just now to actually fail on main, and moved it to the serial branch. I was testing using a modified `ex1p` (hence the screenshots) and committed an incomplete version of the unit test. Should fail on master now (or you can comment out the `bdr_faces` branch in `AccumulateAndCountBdrValues`.